src/BlockTypes/MiniCart.php

```diff
@@ -303,6 +303,11 @@ protected function get_cart_price_markup( $attributes ) {
 		$cart_controller     = $this->get_cart_controller();
 		$cart                = $cart_controller->get_cart_instance();
 		$cart_contents_total = $cart->get_subtotal();
+		
```

Comment:
There are a couple of PHP linting errors because of having white spaces at the end of the file. Here and in line 310.